### DISCUSSION TITLE
DeFi

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#2037] [General]
I am more interested in DefI and want to learn more. I have completed the course, want to thank the team behind such a wonderful course.

Can anyone suggest some resources to deep dive more into the DeFi space !!

**Comment by EngrPips** [comment]
I don't think there is a single place where you can learn everything about Defi, but instead, you get to be more comfortable with Defi by reading different Defi books, taking different Defi courses, and reading and following different Defi blogs. I don't know a bunch of Defi resources at the top of my head, but you can look up resources on Google, I believe.
